charts/agent-control/charts/agent-control-deployment/templates/preinstall-job-register-system-identity.yaml

```diff
@@ -65,7 +65,7 @@ spec:
       serviceAccountName: {{ include "newrelic.common.naming.truncateToDNSWithSuffix" (dict "name" (include "newrelic.common.serviceAccount.name" .) "suffix" "auth" ) }}
       containers:
         - name: register-system-identity
-          image: alpine
+          image: newrelic/agent-control-system-identity-registration:latest
```

Comment:
I think it is better to give the user the possibility to change registry and tag
I know we were not doing it before but it was a mistake.

Sometimes they need to change the registry and "clone" our images